src/App.js

```diff
@@ -0,0 +1,19 @@
+import { Provider } from 'react-redux';
+import BooksList from './components/BooksList';
+import BooksForm from './components/BooksForm';
+import store from './containers/store';
+
+function App() {
+  return (
+    <Provider store={store}>
+      <div className="App">
+        <div>
+          <BooksList />
+          <BooksForm />
+        </div>
+      </div>
```

Comment:
The nested `div` is redundant here. It is better to remove it. ([screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/App/AppWrapBookListAndBookForm.png))
```suggestion
      <div className="App">
        <BooksList />
        <BooksForm />
      </div>
```